iot_verifier/src/gateway_cache.rs

```diff
@@ -65,28 +73,35 @@ impl GatewayCache {
                 Ok(hit.value().clone())
             }
             _ => {
+                tracing::debug!("cache miss: {:?}", address);
```

Comment:
```suggestion
                tracing::debug!(?address, "cache miss");
```
structured log format with the short-hand "assigned log key is the same as the variable name being logged"